pkg/apis/kops/cluster.go

```diff
@@ -132,6 +132,9 @@ type ClusterSpec struct {
 	//   missing: default policy (currently OS security upgrades that do not require a reboot)
 	UpdatePolicy *string `json:"updatePolicy,omitempty"`
 
+	// Use custom IAM roles for cluster roles
+	RoleCustomIamRoles map[string]string `json:"roleCustomIamRoles,omitempty"`
```

Comment:
Yah ... sorry was not clear.  How about the name `CustomIamRoles` or even `IamRoles`?